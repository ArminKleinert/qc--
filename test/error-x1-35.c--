
foo() {
    bits32 x, y;
    bits16 a ,b;

    y = %%add (a,x);
}

