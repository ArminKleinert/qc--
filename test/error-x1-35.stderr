file error-x1-35.c--, line 6, characters 5-21 Error: type error in application
